scout/server/blueprints/cases/views.py

```diff
@@ -579,8 +579,12 @@ def phenotypes_actions(institute_id, case_name):
             # avoid empty lists
             if raw_symbol:
                 # take the first nubmer before |, and remove any space.
-                hgnc_symbol_split = raw_symbol.split('|', 1)[0]
-                hgnc_symbol = int(hgnc_symbol_split.replace(' ', ''))
+                try:
```

Comment:
It works and it's flashing a message when some weird string is provided 👍  
It would also be nice to have an extra check of the input field before user actually submits random strings. I'm thinking about the HTML5 validation: what about adding a basic regex patter there, for instance include in the the <input name="genes" input (line 880 of case.html template) a pattern like `pattern="^[0-9]{4,6}\s\|\s.*"` (feel free to complete the regex, I'm not mastering the art :D. Anyway also this basic check would add some security).

Reply:
I don’t understand at first glance how security would be improved, but I’ll try to read more carefully and see what you mean!

As for crashes and most other random string business, I think this should work.

Reply:
I expressed myself wrong. I meant more that users will see the error immediately. The pattern check will prevent users to even submit wrong strings and display an error instead

Reply:
I mean this:
![image](https://user-images.githubusercontent.com/28093618/69238142-88b8c880-0b97-11ea-978b-d0b4b9dd7288.png)

Reply:
Its a first, and will mess with old browsers. But I do miss that here and some other places. Lets go!